### PR TITLE
Detect interpolation in single quote strings too

### PR DESCRIPTION
**Pull request by silvenon**

### CODE REVIEW

**Comment by 50Wliu** [comment]
This will need specs.

**Comment by silvenon** [comment]
Done.

File: spec/sass-spec.coffee [hunk 1]
```diff
@@ -244,3 +244,39 @@ describe 'SCSS grammar', ->
       expect(tokens[1]).toEqual value: ':', scopes: ['source.css.scss', 'meta.set.variable.scss', 'punctuation.separator.key-value.scss']
       expect(tokens[2]).toEqual value: ' ', scopes: ['source.css.scss', 'meta.set.variable.scss']
       expect(tokens[3]).toEqual value: '$normal-font-size', scopes: ['source.css.scss', 'meta.set.variable.scss', 'variable.scss', 'variable.scss']
+
+  describe 'interpolation', ->
+    it 'is tokenized within single quotes', ->
+      {tokens} = grammar.tokenizeLine 'body { font-family: \'#\{$family}\'; }' # escaping CoffeeScript's interpolation
```
Comment (50Wliu) [inline spec/sass-spec.coffee]: To make this easier you can just switch the outside quotes to double-quotes, and then you won't have to escape the single quotes.
`"body { font-family: '#\{$family}'; }"`

Though why is the second bracket escaped?

**Comment by silvenon** [comment]
Done. Sass and CoffeeScript have the same interpolation syntax, so I'm escaping that bracket to prevent CoffeeScript from interpolating, i.e. CoffeeLint from complaining that I'm trying to interpolate within single quote strings, when CoffeeScript can only interpolate within double quotes.

**Comment by 50Wliu** [comment]
Is there some documentation that dictates that interpolation also works in single-quote strings?  I tried searching for it but can't seem to find anything.

**Comment by silvenon** [comment]
No, it doesn't, but `coffeelint.json` forbids it, I guess to prevent you from doing that thinking that it will work :grin:  Two solutions:

1. keep it that way
2. turn off that CoffeeLint rule

**Comment by 50Wliu** [comment]
No, I meant for Sass :P

**Comment by silvenon** [comment]
Oh. Well, I'm using it and it works :smiley: You can try it on [SassMeister](http://sassmeister.com/) to see for yourself.

**Comment by kevinsawicki** [comment]
Thanks for fixing this :+1:

**Pull request merged**